### PR TITLE
Run terraform fmt

### PR DESCRIPTION
**Pull request by juansanchez-stratusgrid**
## Change description

> Description here

## Type of change
- [x] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [x] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: -inputs.tf [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
 }
 
 variable "description" {
   description = "Description of what your Lambda Function does."
-  type = string
+  type        = string
 }
 
 variable "detect_changes" {
   description = "Controls automatically starting your pipeline when a new commit is made on the configured repository and branch."
```
Comment (angellopez-sg) [inline -inputs.tf]: I think this sentence is wrong, something like "Controls if the codepipeline execution is started automatically when you make a new commit on the repository".

File: iam-codepipeline-lambda.tf [hunk 2]
```diff
@@ -49,19 +49,19 @@ data "aws_iam_policy_document" "lambda_codepipeline" {
...
   }
 }
 
 resource "aws_iam_role_policy" "lambda_codepipeline" {
-  name = "${var.name}_codepipeline_policy"
-  role = aws_iam_role.lambda_codepipeline.id
+  name   = "${var.name}_codepipeline_policy"
+  role   = aws_iam_role.lambda_codepipeline.id
   policy = data.aws_iam_policy_document.lambda_codepipeline.json
 }
```
Comment (angellopez-sg) [inline iam-codepipeline-lambda.tf]: We need a new line here.

File: -inputs.tf (hunk 1 above)
Comment (juansanchez-stratusgrid) [reply -inputs.tf]: Perfect, I upgraded the description of the variable, thanks!

**Review by angellopez-sg — COMMENTED** [review]
LGTM

File: iam-codepipeline-lambda.tf (hunk 2 above)
Comment (juansanchez-stratusgrid) [reply iam-codepipeline-lambda.tf]: Done

**Review by rchildress87 — APPROVED** [review]

**Pull request merged**